msu/hooks/skills/skill_container.nut

```diff
@@ -538,4 +538,19 @@
 
 		return ret;
 	}
+
+// New Events
```

Comment:
This `// New Events` comment is completely out of place here as there are many new events even above this. So I'd just remove this single line.